indico/modules/events/persons/templates/management/person_list.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'events/persons/management/_persons_list_base.html' %}
+
+{% from 'message_box.html' import message_box %}
+
+{% block title %}
+    {% trans %}Event Roles{% endtrans %}
+{% endblock %}
+
+{% block filters %}
+    <div class="group i-selection">
+        <span class="i-button label">{% trans %}Show{% endtrans %}</span>
+        <input type="checkbox" id="filter-secondary" data-filter="chairperson" checked>
+        <label for="filter-secondary" class="i-button">{% trans %}Chairpersons{% endtrans %}</label>
+        <input type="checkbox" id="filter-speakers" data-filter="speaker" checked>
+        <label for="filter-speakers" class="i-button">{% trans %}Speakers{% endtrans %}</label>
+        <input type="checkbox" id="filter-conveners" data-filter="convener" checked>
+        <label for="filter-conveners" class="i-button">{% trans %}Conveners{% endtrans %}</label>
+        <input type="checkbox" id="filter-no-account" data-filter="no_account">
+        <label for="filter-no-account" class="i-button">{% trans %}No account{% endtrans %}</label>
+    </div>
+{% endblock %}
+
+{% block persons %}
+
+    {% if num_no_account %}
+        {% call message_box('warning') %}
+            {{ ngettext("There is one user with no Indico account. An Indico account may be needed to upload materials and/or manage contents.",
+                        "There are {} users with no Indico account. An Indico account may be needed to upload materials and/or manage contents.",
+                        num_no_account).format(num_no_account) }}
+        {% endcall %}
+    {% endif %}
+    <table id="person-list" class="i-table-widget tablesorter">
+        <thead>
+            <tr class="i-table">
+                <th class="i-table checkbox-column" data-sorter="false"></th>
+                <th class="i-table">{% trans %}Name{% endtrans %}</th>
+                <th class="i-table">{% trans %}Email{% endtrans %}</th>
+                <th class="i-table affiliation-column">{% trans %}Affiliation{% endtrans %}</th>
+                <th class="i-table"></th>
+                <th class="i-table"></th>
+                <th class="i-table"></th>
+                <th class="i-table"></th>
+                <th class="i-table"></th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for person_data in persons.itervalues()|sort(attribute='person.full_name') -%}
```

Comment:
I wonder if it'd make more sense to use "Lastname, Firstname" for this table (and also sort accordingly by default)